### PR TITLE
feat: nuke the siteminder session cookie on logout

### PR DESCRIPTION
**Pull request by wenzowski**
According to numerous Rocket Chat threads and https://github.com/bcgov/ocp-sso/issues/4 the logout function only clears the Keycloak session and never clears the SiteMinder SSO session.

This fix attempts to purge the SiteMinder cookie on logout when we're deployed on a subdomain of .gov.bc.ca

### CODE REVIEW

File: app/server/index.js [hunk 1]
```diff
@@ -183,6 +183,16 @@ app.prepare().then(() => {
   };
   const keycloak = new Keycloak({store}, kcConfig);
 
+  // Nuke the siteminder session token on logout if we can
+  // this will be ignored by the user agent unless we're
+  // currently deployed to a subdomain of gov.bc.ca
+  //
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars-experimental
+  server.post('/logout', (req, res, next) => {
```
Comment (matthieu-foucault) [inline app/server/index.js:191]: renaming `req` to `_req` might turn off the eslint error?

Comment (wenzowski) [reply app/server/index.js]: thanks, fixed

**Comment by matthieu-foucault** [comment]
@wenzowski just needs a rebase

**Comment by matthieu-foucault** [comment]
@wenzowski I can confirm this works in the `wksv3k-dev` environment

**Pull request merged**